core/src/main/scala/cromwell/core/eventbus/PubSubMediator.scala

```diff
@@ -0,0 +1,48 @@
+package cromwell.core.eventbus
+
+import akka.actor.{ActorRef, ActorSystem}
+
+final case class EventMsg[A](topic: A, payload: Any)
+
+/**
+  * This is a wrapper to EventStream as one of the event bus with SubChannel trait.
+  */
+trait PubSubMediator {
```

Comment:
As Carlos mentioned in his e-mail, this is there because we will need to use for workflow events logging.